### ISSUE TITLE
gyro_std of new dataset

### ISSUE THREAD

**Issue by DLuminary** [mbrossar/denoise-imu-gyro#8]
Hello Brossar and thanks for sharing your work.

I'm trying to use your method with a different dataset. In order to train and test the network - it seems like gyro_std needs to be defined. 
How can one obtain the values for some arbitrary dataset?

Thanks again!

**Comment by mbrossar** [comment]
Hi, 
this value is used to avoid to have big neural network outputs during the first iterations and thus to train a little bit faster. 

A pretty good value is to take the standard deviation of the gyro error.

**Comment by DLuminary** [comment]
Thanks for your rapid reply @mbrossar 
 
Some questions regarding the main run file - in dataset_params:
1) Does N represent approximately the window size? (in your paper it is 448 and it seems like 500 is used in the code)
2) Do min_train_freq and max_train_freq represent the loss function factors?
3) When using data with 100Hz measurements instead of 200Hz, would you advise lowering the window size and loss factors? What other changes would you suggest?

Best regards

**Comment by RXvo** [comment]
> 感谢您的快速回复@mbrossar
> 
> 关于主运行文件的一些问题 - 在 dataset_params 中：
> 
> 1. N 是否近似代表窗口大小？（在你的论文中它是 448 并且似乎在代码中使用了 500）
> 2. min_train_freq 和 max_train_freq 代表损失函数因子吗？
> 3. 当使用 100Hz 测量而不是 200Hz 的数据时，您会建议降低窗口大小和损耗因子吗？您还建议进行哪些其他更改？
> 
> 最好的祝福

Hello, do you still have the corresponding optimization parameters and data set of this code? The author's links have all failed. Thank you！

**Comment by alun9258** [comment]
> Thanks for your rapid reply @mbrossar
> 
> Some questions regarding the main run file - in dataset_params:
> 
> 1. Does N represent approximately the window size? (in your paper it is 448 and it seems like 500 is used in the code)
> 2. Do min_train_freq and max_train_freq represent the loss function factors?
> 3. When using data with 100Hz measurements instead of 200Hz, would you advise lowering the window size and loss factors? What other changes would you suggest?
> 
> Best regards

Hello, do you still have the corresponding optimization parameters and data set of this code? The author's links have all failed. Thank you！

**Comment by onetree-l** [comment]
> > 感谢您的快速回复
> > 关于主运行文件的一些问题 - dataset_params：
> > 
> > 1. N 代表窗口大小的大致大小吗？（在你的论文中是 448，代码中似乎使用了 500）
> > 2. min_train_freq 和 max_train_freq 是否代表损失函数因子？
> > 3. 当使用具有 100Hz 测量值而不是 200Hz 的数据时，您是否建议减小窗口大小和损耗因子？您还会建议哪些其他更改？
> > 
> > 此致敬意
> 
> 您好，您还有这段代码对应的优化参数和数据集吗？作者的链接都失败了。谢谢！



> 你好
> 此值用于避免在第一次迭代期间出现较大的神经网络输出，从而更快地训练。
> 
> 一个相当不错的值是取陀螺仪误差的标准差。
> 
> En 4 jul2021 7：28， en 7：28， DLuminary ***@***.***> escribió：
> […](#)

您好，您还有这段代码对应的优化参数和数据集吗？作者的链接都失败了。谢谢！
